lektor/markdown.py

```diff
@@ -24,17 +33,19 @@ def link(self, link, title, text):
         title = escape(title)
         return '<a href="%s" title="%s">%s</a>' % (link, title, text)
 
-    def image(self, src, title, text):
+    def image(self, src, alt="", title=None):  # pylint: disable=arguments-differ, arguments-renamed
+        if _old_mistune:
+            (title, alt) = (alt, title)
```

Comment:
As noted above, swapping the semantics of named arguments like this seems dangerous.